### DISCUSSION TITLE
Question about Weekly Repeating Habit and Grey Checkmarks

### DISCUSSION THREAD

**Discussion by papaxanthos** [iSoron/uhabits#705] [Help & FAQ]
First, let me say I've been using this app for a year and it's completely changed my life. Excellent stuff.

Here's the problem I'm facing. I have various habits which repeat 4 or 5 or 6 times a week. My week starts on Monday. If I complete my quota for the week (e.g. 4/4 of a habit), I'd like it to reset on Monday, the beginning of my new week, but this doesn't always happen. Sometimes a grey checkmark will appear for Monday when, in fact, this should be the beginning of the week, and nothing is completed. 

I'm not sure why this is happening. Are habits carrying forward into the next week? For example, if I complete a habit more often than expected (e.g. 5/4 in a week), does that carry forward into the next week? I'd prefer it didn't... Maybe more generally my question is what difference does it make which day of the week is your first? And is there any way to accomplish what I'm trying to do?

**Comment by ghost** [comment]
I think I just proposed the same thing, but it counts as 7 day roll, so if you do it once in a week on Wednesday, it reeds it accomplished until next Tuesday, eventhough you sometimes do it on Wednesday and then next Monday. I just posted the same issue.

------ 

Still having issues with calendar week, even tried to uninstall and reinstall on Monday to try somehow bypass that issue. The example habit is set to once a week, but it is not taking into account calendar week. Is there a way to choose between calendar week and in 7 days, or better X times in a week / X times in 7 days. Perhaps in settings could be chosen the mode it works and shows strikes.

These would be a I think a really needed option.

**Reply by riotpoof** [reply]
I think I'm having the same issue. For me, I like the week to start on a Sunday. I have a goal that I want to complete 5 days/week. Today it is giving me an unfilled checkmark, suggesting that I've already completed this goal 5 times this week. But, it is only the 4th day of this week, so that's impossible. I can't tell if I'm misunderstanding how the app works or if I'm doing something wrong.

Other than that, though, I do love this app! It is really motivating to me. Thanks so much for creating it.
